doc/devel/contributing.rst

```diff
@@ -89,13 +89,13 @@ use on a preexisting environment!
 
 A new environment can be set up with ::
 
-   python3 -mvenv /path/to/devel/env
+   python3 -mvenv <file folder location>
 
 and activated with one of the following::
 
-   source /path/to/devel/env/bin/activate  # Linux/macOS
-   /path/to/devel/env/Scripts/activate.bat  # Windows cmd.exe
-   /path/to/devel/env/Scripts/Activate.ps1  # Windows PowerShell
+   source <file folder location>/activate  # Linux/macOS
+   <file folder location>/Scripts/activate.bat  # Windows cmd.exe
```

Comment:
The windows specific ones should probably have the / go the other way \ as that's what the google style guide recommends and on some dev call we decided that would be a good base reference. https://developers.google.com/style/slashes

Reply:
> @jeromefv the only thing that's holding up approval from me is the slash order for the windows path.

@story645 I'm not sure if I've got that change showing up anywhere else besides my local branch at the moment. When I've executed `git pull upstream master` and see what changes there are, it appears that those edits have already been put in place. I feel like I'm missing something and I'm not quite sure how to move forward from that.